lib/smart_answer_flows/register-a-birth/embassy_result.govspeak.erb

```diff
@@ -0,0 +1,265 @@
+<% content_for :body do %>
+$!You can apply to register the birth with the <%= embassy_high_commission_or_consulate %> in <%= registration_country_name_lowercase_prefix %>.$!
+
+<% if %w(usa).include?(country_of_birth) %>
```

Comment:
Minor: the contents are USA-specific, so it's not likely to have another country use this paragraph, therefore it would be nicer to use `country_of_birth == 'usa'`